### DISCUSSION TITLE
TrakHound-Community Plans

### DISCUSSION THREAD

**Discussion by PatrickRitchie** [TrakHound/TrakHound-Community#46] [General]
I wanted to give an update on what I have planned for this project in the near future.

I will most likely be archiving this repo in the near future and will be replacing it with our new framework. I'm hoping to release the new framework before the end of this year and will look to have a new simple dashboard app available around Spring of next year. 

I had meant to update/replace this project a while ago but never found a good solution that would allow me to use the same system(s) for a simple 24hr dashboard as I would use for a more advanced historical system. Over the past few years I've developed a brand new framework from scratch that will accomplish this and I should have it released by the end of this year. I meant to have it released this summer but there were some things that weren't quite right and so I delayed the release.

For machine monitoring in general, I believe the trend now is for Machine Tool OEMs to offer their own dashboard monitor apps that are similar in function to what I was doing with TrakHound-Community which I think makes sense. Creating a fairly simple dashboard is not too time intensive for a small developer team to develop and offers the OEM an "Industry 4.0 solution" to show off how their machine can be monitored remotely. The complicated part is how you take that data and integrate it into a larger system with other types of data. That is where my new framework will hopefully play a major role.

I like offering a free and self-contained app in order to provide an example of how machine data can be utilized so I will look to offer a similar app using the new framework. I think this helps users who are curious to know what value monitoring can bring as well as for applications that may only require a simple status dashboard.

Thank you to everyone who has supported this project and I look forward to introducing all of the new upcoming projects to everyone. Hopefully this project has been helpful and has been a first step for manufacturers to start putting value on their machine data and get them thinking about future use cases.

Thanks,
-Patrick

**Comment by ltomuta** [comment]
"I like offering a free and self-contained app in order to provide an example of how machine data can be utilized so I will look to offer a similar app using the new framework. I think this helps users who are curious to know what value monitoring can bring as well as for applications that may only require a simple status dashboard.'

Indeed, something like that is definitely needed. I've been scouring the interwebs for a simple app that would just get us starting to look into the possibilities of tapping into a machine's data.  I could not believe that all I found was manufacturer specific and/or ghost traces of past projects.  Until recently we didn't even know that our machines can give us so much data through a standardized protocol.

Thanks for your work on this project and I'm looking forward to the next one.

**Reply by PatrickRitchie** [reply]
Thanks for the feedback and I'm glad you have found this project useful. Yeah I think a simple app is very valuable as that can often be a proof of concept that engineers use to present to management on how useful a larger data solution can be. 

If you are new to MTConnect, I would suggest looking at my other project [MTConnect.NET](https://github.com/TrakHound/MTConnect.NET) that has tools for implementing MTConnect Clients, Agents, and Adapters.
